### PR TITLE
Fixed link to docs

### LINKED ISSUE #3445: Documentation link points to Docusaurus
On https://telescope.cdot.systems/docs/ in the bottom-right, the Documentation link is pointing to Docusaurus.  Let's point back to our docs:

Screen Shot 2022-04-08 at 7 12 32 PM

### PR DESCRIPTION
**Pull request by mnosov622**
Hello!

Please review the bug fix.

## Issue This PR Addresses

1. Automatically close the issue when this PR is merged
    USAGE: Fixes #3445 

## Type of Change

**Bug** 

## Description

Fixed the link to the Telescope documentation, the previous link was going to docusaurus documentaton

## Checklist

- [ ] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: 
screenshot

- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

File: src/web/docusaurus/docusaurus.config.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ const config = async () => {
               items: [
                 {
                   label: 'Documentation',
-                  href: 'https://docusaurus.io/docs/',
+                  href: 'https://telescope.cdot.systems/docs/overview/',
```
Comment (Kevan-Y) [inline src/web/docusaurus/docusaurus.config.js:123]: We have 2 environments for this 
One staging: https://dev.telescope.cdot.systems/docs/overview/
One production: https://telescope.cdot.systems/docs/overview/

I don't think it's a good idea to redirect always to production.

I believe you can do relatives path [see](https://docusaurus.io/docs/api/themes/configuration#footer-links)

**Review by Kevan-Y — CHANGES_REQUESTED** [review]

File: src/web/docusaurus/docusaurus.config.js (hunk 1 above)
Comment (mnosov622) [reply src/web/docusaurus/docusaurus.config.js]: Hi @Kevan-Y ,

Thank you for your feedback, I changed the link, hopefully it's okay :)

Comment (humphd) [reply src/web/docusaurus/docusaurus.config.js]: @mnosov622 it looks like you've switched to using an absolute URL to staging now, which also won't work.  What @Kevan-Y is saying is that we need this to work no matter where we host this site.  The link needs to take into account the fact that we host this same site on different URLs.

Comment (mnosov622) [reply src/web/docusaurus/docusaurus.config.js]: Hi @humphd ,

Oh, yes, I'm sorry, i get it. Please let me know if everything is fine now :)

File: src/web/docusaurus/docusaurus.config.js [hunk 2]
```diff
@@ -120,7 +120,7 @@ const config = async () => {
               items: [
                 {
                   label: 'Documentation',
-                  href: 'https://docusaurus.io/docs/',
+                  href: '/docs/overview',
```
Comment (Kevan-Y) [inline src/web/docusaurus/docusaurus.config.js:123]:
```suggestion
                  to: '/docs/overview',
```

**Review by Kevan-Y — CHANGES_REQUESTED** [review]

File: src/web/docusaurus/docusaurus.config.js (hunk 2 above)
Comment (Kevan-Y) [reply src/web/docusaurus/docusaurus.config.js:123]: Based on doc `to` is preferable for internal navigation

Comment (mnosov622) [reply src/web/docusaurus/docusaurus.config.js]: Yes, thank you @Kevan-Y .

**Review by Kevan-Y — APPROVED** [review]

**Review by cindyledev — CHANGES_REQUESTED** [review]
Thanks a lot for doing this. Please squash and rebase your commits then we can have it merged into the master branch :)

https://telescope.cdot.systems/docs/contributing/git-workflow

**Comment by mnosov622** [comment]
Hello!

I'm sorry, I will make another pull request, hopefully it's okay :)

**Comment by mnosov622** [comment]
I created another pull request, thank you :) #3674

**Comment by Kevan-Y** [comment]
> Thanks a lot for doing this. Please squash and rebase your commits then we can have it merged into the master branch :)
> 
> https://telescope.cdot.systems/docs/contributing/git-workflow

GitHub now has squash and merge button (I use too much of this button😅), but we just need to make sure the branch has the latest stuff
image

**Comment by mnosov622** [comment]
Thank you, Kevan ! I didn't know about that :)

**Pull request closed without merging**